frontend/src/styles/HetComponents/HetLaunchLink.tsx

```diff
@@ -0,0 +1,16 @@
+import type React from 'react'
+import { LaunchRounded } from '@mui/icons-material'
+
+interface HetLaunchLinkProps {
+  href: string
+}
+
+const HetLaunchLink: React.FC<HetLaunchLinkProps> = ({ href }) => {
+  return (
+    <a href={href} target='_blank' rel='noreferrer'>
+        <LaunchRounded className='text-text' />
```

Comment:
i think this needs an `aria-label` since the icon doesn't have any text label with it. I don't think the label has to be the text title it's next to, just something like "open link in a new window" or something